types/testingutils/network.go

```diff
@@ -20,7 +20,7 @@ func NewTestingNetwork(operatorID types.OperatorID, sk *rsa.PrivateKey) *Testing
 	}
 }
 
-func (net *TestingNetwork) Broadcast(message *types.SignedSSVMessage) error {
+func (net *TestingNetwork) Broadcast(msgID types.MessageID, message *types.SignedSSVMessage) error {
```

Comment:
I am thinking about this further now...
you only need the `ValidatorPublicKey`?
So maybe it is best to simply pass `recipientID []byte` parameter for topic.

This is because in the very near future (cluster-based consensus), we will be using `ClusterID` exclusively for topic calculation (at least this is the current plan), while it may not necessarily be in the `MessageID` (duty dependent). So my offer will save a future change...

Your choice whether you want to change it now of course